### PR TITLE
write_redis: Added option to choose redis database to use

### PR DESCRIPTION
**Pull request by eryx12o45**

### CODE REVIEW

File: src/write_redis.c [hunk 1]
```diff
@@ -45,6 +49,7 @@ struct wr_node_s
   int port;
   struct timeval timeout;
   char *prefix;
+  char *database;
```
Comment (octo) [inline src/write_redis.c]: If I read the docs correctly, this is a numeric id. Would it make sense to use an `int` here (and `cf_util_get_int()` below)?

File: src/collectd.conf.pod [hunk 2]
```diff
@@ -7503,7 +7504,8 @@ the timestamp as the score. Retrieving a date range can then be done using the
 C<ZRANGEBYSCORE> I<Redis> command. Additionnally, all the identifiers of these
 I<Sorted Sets> are kept in a I<Set> called C<collectd/values> (or
 C<${prefix}/values> if the B<Prefix> option was specified) and can be retrieved
-using the C<SMEMBERS> I<Redis> command. See
+using the C<SMEMBERS> I<Redis> command. You can specify the database to use 
+with the Database parameter (default is 0). See
 L<http://redis.io/commands#sorted_set> and L<http://redis.io/commands#set> for
 details.
```
Comment (octo) [inline src/collectd.conf.pod]: Can you please add a more detailed description below? Something like:

```
=item B<Database> I<Index>

Selects the database … Defaults to C<0>.
```

See the *perlpod(1)* manpage for a description of the syntax.

**Comment by octo** [comment]
Hi @eryx12o45! Thank you very much for your patch! I've left some review comments inline, when they're resolved this is good to go in.

Thanks and best regards,
—octo

**Comment by eryx12o45** [comment]
HI octo,

I've changed the code according to your requests. Hope this is what you've had in mind.

Best regards
- eryx

**Comment by octo** [comment]
Awesome, thanks @eryx12o45!

**Pull request merged**